utils/utils.c

```diff
@@ -392,3 +392,81 @@ char * strjoin(char *left, char *right, char *delim)
 	strcpy(new + len - rlen - 1, right);
 	return new;
 }
+
+#define QUOTE '\''
+#define DQUOTE '"'
+
+static int setargs(char *args, char **argv)
```

Comment:
This function is little bit complex so it'd be nice if you could add some comment here.